src/dot-components/Cart/cart.css

```diff
@@ -0,0 +1,98 @@
+@import url("https://techtonic-ui.netlify.app/styles.css");
```

Comment:
I dont think this is import is needed as you have already added it in header of your index.html

Reply:
Yeah, forgot to remove it.